app/api/account.py

```diff
@@ -0,0 +1,15 @@
+from fastapi import APIRouter, status, Depends
+from models.models import Account, AccountUpdate
+from service.auth import get_current_user
+from service.account import create_account, update_account
+
+router = APIRouter()
+
+@router.post('/account', status_code=status.HTTP_201_CREATED)
```

Comment:
```suggestion
@router.post('/accounts', status_code=status.HTTP_201_CREATED)
```
p1; REST에서 Collection에 해당하는 부분은 복수형을 써주시는게 좋습니다. 아래의 users는 잘 써주셨는데 일관성있게 맞춰주시면 좋을것 같습니다.

Reply:
네! 감사합니다! 수정하였습니다!